### PR TITLE
create chart and display data

### PR DESCRIPTION
**Pull request by daytonbrock**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
   "dependencies": {
     "axios": "^0.19.0",
     "bcryptjs": "^2.4.3",
+    "chart.js": "^2.9.3",
```
Comment (mkbaker) [inline package.json:9]: Wondering if this can be removed? It doesn't seem to be used

Comment (mkbaker) [inline package.json:9]: Can this be removed? It doesn't seem to be used

**Review by mkbaker — APPROVED** [review]
Very cool!

File: package.json (hunk 1 above)
Comment (daytonbrock) [reply package.json:9]: the readme for react-chart-js2 said that in recent updates they made chart.js a peer dependency, that it should be added to the dependencies. I didn't see it in the code you sent me though, so I wasn't sure.

Comment (mkbaker) [reply package.json:9]: Gotcha!

**Pull request merged**